test/test_onnx.py

```diff
@@ -1,13 +1,11 @@
-# onnxruntime requires python 3.5 or above
-try:
-    # This import should be before that of torch
-    # see https://github.com/onnx/onnx/issues/2394#issuecomment-581638840
-    import onnxruntime
-except ImportError:
-    onnxruntime = None
-
 from common_utils import set_rng_seed, assert_equal
 import io
+import pytest
+# In environments without onnxruntime we prefer to
+# invoke all tests in the repo and have this one skipped rather than fail.
+# This import should be before that of torch
+# see https://github.com/onnx/onnx/issues/2394#issuecomment-581638840
```

Comment:
Thanks @garymm , looks good. Do you know if this comment is still relevant? The original issue was closed so I feel like we should be able to remove the comment and do that `importskip` lower down, where it will be easier to find.

Do you mind trying that, and if there are still issue, we can put it back?

Reply:
Thanks for checking this. Done.